### ISSUE TITLE
[Bug] File not allowed error

### ISSUE THREAD

**Issue by Enferlain** [nonnonstop/sd-webui-3d-open-pose-editor#84]
**Describe the bug**
Go to 3d openpose and see this

![image](https://github.com/nonnonstop/sd-webui-3d-open-pose-editor/assets/15861396/aafc9a87-68db-4664-8935-327e439d7f52)

![image](https://github.com/nonnonstop/sd-webui-3d-open-pose-editor/assets/15861396/7f63f760-3e74-4791-815a-28ac03011c65)

**To Reproduce**
Clone to a drive folder. Share it to a different drive account, launch with a colab notebook from the account it was shared to

**Expected behavior**
Prior to 1.3.2 it worked without problems, it seems to be related to some new change

**Environment (please complete the following information):**
 - OS: Linux
 - Browser: Chrome 
 - A1111 WebUI Version: [baf6946](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/baf6946e06249c5af9851c60171692c44ef633e0)
 - Extension Version: (https://github.com/nonnonstop/sd-webui-3d-open-pose-editor/commit/f2d5aac51d891bc5f266b1549f3cf4495fc52160)
 - ControlNet Version: (https://github.com/Mikubill/sd-webui-controlnet/commit/f36493878b299c367bc51f2935fd7e6d19188569)

**Additional context**
Basically it's a shared drive environment. On the base account (the one sharing the folders) it's a nonissue, it only appears on the accounts that launch via shortcuts of the shared folders. For now everything else other than this and style.css access works.

**Comment by MalfreCryvertia** [comment]
ive got the same error, how do u fix this?

**Comment by Enferlain** [comment]
Reinstalled and updated webui and updated to the latest version of the extension. I had this problem in a patchwork colab notebook setup, it hasn't appeared ever since.
